setup.py

```diff
@@ -240,10 +257,19 @@ def run(self):
     download_url="https://github.com/rdiff-backup/rdiff-backup/releases",
     python_requires='~=3.6',
     platforms=['linux', 'win32'],
+    entry_points={
+        'console_scripts': [
+            'rdiff-backup = rdiffbackup.run:main',
+            'rdiff-backup-delete = rdiff_backup.run_delete:main',
+            'rdiff-backup-statistics = rdiff_backup.run_stats:main',
+        ]
+    },
     packages=["rdiff_backup", "rdiffbackup",
               "rdiffbackup.actions", "rdiffbackup.utils", "rdiffbackup.meta",
               "rdiffbackup.locations", "rdiffbackup.locations.map"],
     package_dir={"": "src"},  # tell distutils packages are under src
+    include_package_data=True,
+    package_data={"rdiff_backup": ["*.dll"]},
```

Comment:
Would that include the dll file for Linux too !?

Reply:
No, it just wouldn't exist. And that would simply be ignored.